pkg/BUILD

```diff
@@ -42,6 +42,15 @@ py_binary(
     ],
 )
 
+py_binary(
+    name = "build_zip",
+    srcs = ["build_zip.py"],
+    visibility = ["//visibility:public"],
+    deps = [
+        ":archive",
+        "@abseil_py//absl/flags",
```

Comment:
I saw in #49 that you want to get rid of this dependency. I wrote this mirroring the `pkg_tar` code. Would you prefer it using abseil for now or immediately going to `optparse`?

Reply:
Go to optparse. It means less change in the future.

Reply:
Done!